doc/usermanual.md

```diff
@@ -609,78 +609,48 @@ Stores the information about the IoTAgent for further use in the `retrieveVersio
 All contributions to this project are welcome. Developers planning to contribute should follow the [Contribution Guidelines](./docs/contribution.md)
 
 ### Project build
-The project is managed using Grunt Task Runner.
```

Comment:
Looking to the tasks defined in packages.json:

    "clean": "...",
    "test": "...",
    "test:watch": "...",
    "lint": "...",
    "test:coverage": "...",
    "watch:lib": "..."

I miss "clean" and "test:watch" to be described in this documentation.

Reply:
Fixed with 139d5b8ce4c3f7aa435921043289e7d235f8c5d7